eslint.config.mjs

```diff
@@ -18,7 +18,7 @@ export default [...compat.extends('@scality/scality'), {
     },
 
     languageOptions: {
-        ecmaVersion: 2020,
+        ecmaVersion: 2021,
```

Comment:
Can we use the latest or some parts are not yet compatible (and should probably be identified)?